build.sh

```diff
@@ -300,7 +300,6 @@ if buildAll || hasArg libcudf; then
     # Record build times
     if [[ "$BUILD_REPORT_METRICS" == "ON" && -f "${LIB_BUILD_DIR}/.ninja_log" ]]; then
         echo "Formatting build metrics"
-        python ${REPODIR}/cpp/scripts/sort_ninja_log.py ${LIB_BUILD_DIR}/.ninja_log --fmt xml > ${LIB_BUILD_DIR}/ninja_log.xml
         MSG="<p>"
```

Comment:
Does this tag get closed? It also appears to repeat the tag in the output: `print("<p>", msg, "</p>")`

Maybe it just needs to start as an empty string.
```suggestion
        MSG=""
```